scripts/deploy/deploy.js

```diff
@@ -27,20 +30,23 @@ const AWS_CONFIG = {
 /**
  * Deploy SDK files to CDN
  * Usage:
- * node deploy.js staging|prod staging|canary|vXXX root,us1,eu1,...
+ * node deploy.js staging|prod staging|canary|pull-request|vXXX root,pull-Request,us1,eu1,...
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏no?
```suggestion
 * node deploy.js staging|prod staging|canary|pull-request|vXXX root,pull-request,us1,eu1,...
```